crates/api_common/src/person.rs

```diff
@@ -441,3 +441,14 @@ pub struct ListMedia {
 pub struct ListMediaResponse {
   pub images: Vec<LocalImageView>,
 }
+
+#[derive(Debug, Deserialize)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+pub struct GetVoteAnalyticsByPerson {
+  pub person_id: PersonId,
+  pub exclude_votes_on_self: Option<bool>,
```

Comment:
Users always upvote their own posts and comments by default. So this option seems useless, instead votes on self should always be excluded.